packages/docusaurus/src/client/templates/ssr.html.template.js

```diff
@@ -35,6 +35,10 @@ module.exports = `
       <script type="text/javascript" src="<%= it.baseUrl %><%= script %>"></script>
     <% }); %>
     <%~ it.postBodyTags %>
+    <script>
+      document.getElementById('css-didnt-load-baseurl-suggestion').innerHTML =
```

Comment:
We'd rather be "defensive" and consider the element might not be found, in such case we should not have any console error